src/src/user/user.service.ts

```diff
@@ -870,6 +871,14 @@ export class UserService {
 		}
 
 		if (resp) {
+			if (mappedResponse.keycloak_id) {
```

Comment:
change condition to
`if (!mappedResponse.username && mappedResponse.keycloak_id)`

Reply:
changes done